index.php

```diff
@@ -8,17 +8,48 @@
 
 $plants = getData($db);
 
+$images = getImage($db);
+
+$plantImageArray = displayImage($images);
+
+$ids = getID($db);
+$id = displayID($ids);
+
 ?>
 
 <!DOCTYPE html>
 <html lang="eng">
 <head>
     <meta charset="utf-8">
     <title>Collectors App</title>
-    <link rel="stylesheet" type="text/css" href="index.css">
-    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="styles/index.css">
+    <link rel="stylesheet" type="text/css" href="styles/normalize.css">
```

Comment:
Normalize should come before your own CSS, because the browser will overwrite rules in downwards-order. You want your own rules to override normalize, not the other way around.